webapp/src/ts/components/analytics-targets-details/analytics-targets-details.component.html

```diff
@@ -0,0 +1,13 @@
+<span *ngIf="!value?.placeholder; else valuePlaceholder">
+  <span *ngIf="!target?.isPercent; else targetIsPercent">{{ value?.pass }}</span>
+  <ng-template #targetIsPercent>
```

Comment:
I did this for some simple and "less important" template's content, I left the ones that are already using variables of higher (parent) scope in the original place and to keep it simple for now, I won't use template-outlet for these since they're small. :)